assets/queries/k8s/statefulsets_headless_service_name/query.rego

```diff
@@ -0,0 +1,21 @@
+package Cx
+
+CxPolicy [ result ] {
+  metadata := input.document[i].metadata
+  input.document[i].kind == "Service"
+  specs := input.document[i].spec
+  specs.clusterIP != "None"
+  some j
+  input.document[j].kind == "StatefulSet"
+  input.document[j].spec.selector.matchLabels == input.document[j].spec.template.metadata.labels
+  metadata.name == input.document[j].spec.serviceName
+
+  
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("metadata.name=%s.clusterIP", [metadata.name]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": sprintf("metadata.name=%s.clusterIP creates a Headless Service", [metadata.name]),
+                "keyActualValue": 	sprintf("metadata.name=%s.clusterIP doesn't creates a Headless Service", [metadata.name]),
```

Comment:
clusterIP is part of the Service resource, not the StatefulSet, you should say something like:
"'metadata.name=%s.spec.serviceName' refers to a Headless Service"
and the negation in the keyActualValue